cmd/admin/cert.go

```diff
@@ -297,6 +298,7 @@ func (a *admin) revokeSerials(ctx context.Context, serials []string, reason revo
 					},
 				)
 				if err != nil {
+					anyErr = true
```

Comment:
It's not racy because none of these goroutines read this value: if more than one of them tries to set it to true at the same time, doing so is idempotent. But yes if we wanted to count the number of errors, then we'd need to use a mutex or an atomic int. Would you prefer that we use a counter so the final error message can be more precise?

Reply:
I don't think an error count is necessary, so let's pretend I said `anyErr` should be atomic.Bool and we do `anyErr.Store(true)` instead.

The Go data race detector does fire on a similar test program I just wrote.

Reply:
```
package main

import (
        "fmt"
        "sync"
)

func main() {
        found := false
        wg := new(sync.WaitGroup)
        for x := range 100 {
                wg.Add(1)
                go func(i int) {
                        defer wg.Done()
                        if i%7 == 0 {
                                found = true
                        }
                }(x)
        }
        wg.Wait()
        fmt.Println(found)
}
```

Reply:
Ah I see what you're saying. Yes, this is a data race as far as the race detector is concerned, because it defines a race as "when two goroutines access the same variable concurrently and at least one of the accesses is a write", and we do have multiple goroutines writing to the same variable.

It's not capable of detecting that those writes are idempotent, as we know these ones are. It's unfortunate that the race detector can't detect (or use static analysis to prove) that the value being written is constant.

I've changed this to an atomic.Uint64 (if we're using atomics anyway, might as well take advantage of it) so that the race detector doesn't flakily fire during CI runs.

Reply:
I agree it's (probably!) fine, but unless there's a strong need, I think it is worth ensuring we don't trip the race detector :)